connectors/codemaat.py

```diff
@@ -172,7 +172,10 @@ def ownership_patterns(self, git_log_file):
         # Insert the patterns of ownership into the database
         for index, row in df.iterrows():
             # Create the author if it doesn't exist
-            author = self.session.query(Author).filter(Author.name == row['author']).first()
+            author = self.session.query(Author, Ownership) \
+                    .join(Ownership, Author.author_id == Ownership.author_id) \
+                    .filter(Author.name == row['author']) \
+                    .filter(Ownership.version_id == Version.version_id).first()
```

Comment:
This can't wok as Version.version_id is used as a kind of unset static field

Reply:
I fix it